schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT,
```

Comment:
### Optional
- [ ] I suggest that you make the ID column not null and autoincremented
like `id int NOT NULL IDENTITY`
- [ ] Mark the id column as primary key
`PRIMARY KEY (id)`

Reply:
Thanks for reminding me about these issues! I used BIGSERIAL(BIGINT) for generating ids.